### DISCUSSION TITLE
Can I define colortheme with RGB code by passing them from command options?

### DISCUSSION THREAD

**Discussion by linrongbin16** [sharkdp/bat#2645] [Q&A] [answered]
Hi,

fzf support define color themes by using: `bg:#74f630`, `fg`, `bg+` something like this.

It works a little bit like vim syntax highlighting.

Does bat support something like that?

**Comment by keith-hall** [comment, marked as answer]
Currently this is unsupported, but there is an open feature request for it which is waiting for someone to implement: https://github.com/sharkdp/bat/issues/339

**Reply by linrongbin16** [reply]
got it thanks
